db/migrate/20200930115329_add_record_type_to_trainees.rb

```diff
@@ -0,0 +1,5 @@
+class AddRecordTypeToTrainees < ActiveRecord::Migration[6.0]
+  def change
+    add_column :trainees, :record_type, :integer
```

Comment:
This should have an index

Reply:
I've changed the migration to include adding an index